src/test/java/Homework22.java

```diff
@@ -0,0 +1,54 @@
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+
+import org.testng.Assert;
+import com.beust.jcommander.Parameter;
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.BeforeSuite;
+import org.testng.annotations.Test;
+import org.openqa.selenium.WebElement;
+import pages.AllSongsPage;
+import pages.HomePage;
+import pages.LoginPage;
+
+import java.time.Duration;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class Homework22 extends BaseTest {
+
+    LoginPage loginPage = new LoginPage(driver);
+    HomePage homePage = new HomePage(driver);
+    AllSongsPage allSongPage = new AllSongsPage(driver);
+    @Test
+    public void createPlaylistAndAddSong() {
+        //<--login-->
+        setupClass();
+        loginPage.loginPageTest();
+
+
+        //<--create playlist/add song/delete playlist-->
+        homePage.createPlaylist();
+        allSongPage.addSong();
+        homePage.deletePlaylist();
```

Comment:
Please, use in test methods, the following logical split:   GIVEN – WHEN - THEN

For example

![Clean_Test](https://user-images.githubusercontent.com/104761482/224950509-f0164347-1ab8-40c5-8259-7cfede92912d.png)